README.md

```diff
@@ -607,11 +633,33 @@ Name | Type |
 ------ | ------ |
 `value` | any |
 
+###  isUrl
+
+• **isUrl**: *function*
+
+*Defined in [validators/validators.ts:63](https://github.com/openmrs/openmrs-esm-module-config/blob/master/src/validators/validators.ts#L63)*
+
+#### Type declaration:
+
+▸ (`value`: any): *void | String*
+
+**Parameters:**
+
+Name | Type |
```

Comment:
These are generated, too, right? Because the syntax is a bit off, missing the initial pipe.